src/constants/settings.ts

```diff
@@ -1,7 +1,6 @@
 import StellarSdk from "stellar-sdk";
 
-// TODO: set to 100 before launch
-export const TX_HISTORY_LIMIT = 20;
+export const TX_HISTORY_LIMIT = 100;
```

Comment:
❓ Is this related to resolving a federation address, or accidental change? Looks unrelated to me.

Reply:
@leighmcculloch It's unrelated to resolving the federation address. I just snuck it in there because it was so tiny. :blush: